jwst/tso_photometry/tso_photometry.py

```diff
@@ -59,6 +60,15 @@ def tso_aperture_photometry(datamodel, xcenter, ycenter, radius, radius_inner,
         bkg_aper = CircularAnnulus((xcenter, ycenter), r_in=radius_inner,
                                    r_out=radius_outer)
 
+    # convert the input data and errors from MJy/sr to Jy
+    if datamodel.meta.bunit_data != 'MJy/sr':
```

Comment:
Probably want to add a test for existence here, in case the BUNIT keyword isn't in the header.

Reply:
done

Reply:
I don't see a code change. Or has the updated version not been committed yet?

Reply:
pushed now, thanks.

Reply:
What we have here is failure to communicate ... when I said "add a test for existence" I was referring to a check in the code to make sure that `datamodel.meta.bunit_data` actually exists, before trying to do a test on its value, not the addition of a unit test for the case where the keyword doesn't exist (although the latter doesn't hurt either). I'm just worried that the step itself is going to crash with some kind of weird NoneType error if `datamodel.meta.bunit_data` isn't defined or has the value `None`. So check that the attribute `is not None` before doing the value comparison.

Reply:
I did both.  Is this not what you were looking for:  https://github.com/spacetelescope/jwst/pull/5529/files#diff-486ca4d2c6af260ec03cb90061a92b296db28d424ccb5904b1f11f69f07349fdR40-R43

Reply:
https://github.com/spacetelescope/jwst/pull/5529/commits/0cf23cef8fa62ab6d943bc3d6a226db65d2c5127

Reply:
D'oh! Yep, that's it. I was looking for it in this module and didn't see it in the step module. Never mind ...